mcp-tools/maos-mcp-hub/lib/jira/client.py

```
@@ -334,6 +334,226 @@ async def set_estimation(self, issue_key: str, board_id: int, value: float) -> d
             max_retries=0,  # PUT is non-idempotent for estimation
         )
 
+    # ========================================================================
+    # Issues CRUD — create, edit, transition, search
+    # ========================================================================
+
+    async def create_issue(
+        self, project_key: str, issue_type: str, summary: str, **fields
```

Ruff (0.15.9)

[warning] Missing type annotation for `**fields`

(ANN003)